Makefile

```diff
@@ -2512,7 +2613,21 @@ endif
 	    $(ocamlopt_CMO_FILES) \
 	    "$(INSTALL_COMPLIBDIR)"
 ifeq "$(build_ocamldoc)" "true"
-	$(MAKE) -C ocamldoc installopt
+	$(MKDIR) "$(INSTALL_BINDIR)"
```

Comment:
Similarly (although `installopt` only implicitly assumes that `install` has already been called)